errolyn/a10.py

```diff
@@ -0,0 +1,35 @@
+
+bit_string = input("Please enter a 32-bit integer: ") # example: 10000000111111110000001100001111
+# there is no input sanitization or error handling since the assignment did not ask for it,
+# I would turn this into it's own function if that was a requirement.
```

Comment:
We should still consider making these parameters instead of global variables. `parce_8_bit_string` not taking any arguments makes it pretty unclear what variables it modifies.